integration-tests/android.spec-int.js

```diff
@@ -65,6 +65,41 @@ test.describe('Protections toggle', () => {
         await page.waitForTimeout(500) // todo(Shane): remove this
         await dash.mocks.calledForToggleAllowList()
     })
+    test('with alternative primary screen - toggling protections', async ({ page }) => {
+        const dash = await DashboardPage.android(page)
+        await dash.addState([testDataStates['alternative-layout-exp-1']])
+        await dash.showsAlternativeLayout()
+        await dash.toggleProtectionsOff()
+        await page.waitForTimeout(500) // todo(Shane): remove this
```

Comment:
❓  Does this need removed?

Reply:
Not yet - the messaging mocks are not smart enough yet :(